grakn-factory/janus-factory/src/main/resources/default-configs.properties

```diff
@@ -0,0 +1,75 @@
+#
+# Grakn - A Distributed Semantic Database
+# Copyright (C) 2016  Grakn Labs Limited
+#
+# Grakn is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Grakn is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+#
+
+# This provides keeps a variety of configs which should not be exposed or configurable to the user,
+# but are still required when initialising Janus Based DBs.
+# Note: That these default configs can be overwritten at runtime
+
+##############################################################################################
+############################### Cassandra Configuration ######################################
+##############################################################################################
+
+# Cassandra thrift frame size in megabytes
+storage.cassandra.frame-size-mb=200
+
+##############################################################################################
+############################# Configuration Needed For Analytics #############################
+##############################################################################################
+
+# Janus InputFormat configuration for using hadoop
+# Not clear why these need to be set. See http://stackoverflow.com/questions/38524151/counting-vertices-on-a-titan-graph-using-sparkgraphcomputer-throws-org-apache-sp/38529076
+# for more information.
+# These two properties should be equal to the value of storage.backend and storage.hostname, respectively.
+janusmr.ioformat.conf.storage.backend=cassandra
+janusmr.ioformat.conf.storage.hostname=localhost
```

Comment:
This needs to be the same as `storage.hostname` in grakn.properties.
Is there a way to capture `storage.hostname` and pass it to the layer below?
In any other scenario aside from running Grakn on your local PC, you will need to change this setting as well which makes the whole exercise moot.

Reply:
Ahhh good spot you are correct. We can map and inject the values from the following:

```
storage.backend=grakn-production
storage.hostname=127.0.0.1
```

It will be weird but it will work.